README.md

```diff
@@ -1,2 +1,194 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- An npm installed on your syatem.
+
+#### Windows
+
+[Visit the link to download the package](https://nodejs.org/dist/v18.15.0/node-v18.15.0-x86.msi)
+
+#### Linux
+
+```sh
+ sudo apt install nodejs
+```
+
+#### MacOS
+
+[Visit the link to download the package](https://nodejs.org/dist/v18.15.0/node-v18.15.0.pkg)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/calebchris000/Space-Traveler-s-Hub.git && cd Space-Traveler-s-Hub
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Running Tests
+
+To to test the project, checkout [Jest](https://jestjs.io/)
+
+```sh
+npm install jest --save-dev
+```
+Then, to start the test:
+
+
+```sh
+  npm run test
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+### Deployment
+
+The project has been deployed with Github pages. You can visit the link [here](https://calebchris000.github.io/Bookstore/build)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Caleb Nwaizu**
+
+- GitHub: [@githubhandle](https://github.com/calebchris000)
+- Twitter: [@twitterhandle](https://twitter.com/calebchris000)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/caleb-nwaizu-b815aa23b/)
+
+👤 **Emmanuel Obi**
+
+- GitHub: [@githubhandle](https://github.com/Gmanlove)
+- Angelist: [Angelist](https://wellfound.com/u/emmanuel-obi-9)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emmanuelgozieobi/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[UI enhancemets is coming soon.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it s a star :star:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the people involved for making this project possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional